mem_blocks.go

```diff
@@ -29,6 +30,7 @@ import (
 type MemoryBlockIterator interface {
 	First() *MemoryBlock
 	Next() *MemoryBlock
+	Filesize() uint64
```

Comment:
Iterators are not required to implement a file size function (`iterator->file_size` may be `NULL`), so this should go into a separate optional interface.

Reply:
Done